app/assets/javascripts/govuk_publishing_components/analytics/explicit-cross-domain-links.js

```diff
@@ -0,0 +1,43 @@
+/* global GOVUK, $, ga */
+
+(function () {
+  'use strict'
+  window.GOVUK = window.GOVUK || {}
+
+  var ExplicitCrossDomainLinks = function () {
+    this.decorateForm = function (element, trackers) {
+      if (trackers.length) {
+        var existing = element.attr('action')
```

Comment:
We're trying to remove our dependency on jQuery so all new JS code is supposed to be written without it. Having said that, that's a bit of an ask. If you like you can stick a card in the backlog to get this de-jQuery-ed as a piece of tech debt, and I can look at it later.

Reply:
(related - tests don't need to have jQuery removed as jasmine has its own jQuery for running tests, independent of the publically-facing frontend code)